packages/js/src/components/TimestampToggle.js

```diff
@@ -0,0 +1,163 @@
+import {Component} from '@wordpress/element';
+import PropTypes from "prop-types";
+import styled from "styled-components";
```

Comment:
We're moving away from `styled-components`, could you add styling through a `className` instead?